hack/validate-ci-secret-boostrap.sh

```
@@ -7,7 +7,7 @@ cd $(dirname $0)/..
 BOOTSTRAP_BINARY=${BOOTSTRAP_BINARY:-/usr/bin/ci-secret-bootstrap}
 
 if [[ ! -x ${BOOTSTRAP_BINARY} ]]; then
-  cd ../ci-tools && go build -race=true -o ${BOOTSTRAP_BINARY} ./cmd/ci-secret-bootstrap && cd -
+  cd ../ci-tools && go build -mod=mod -race=true -o ${BOOTSTRAP_BINARY} ./cmd/ci-secret-bootstrap && cd -
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)